packages/dds/tree/src/test/feature-libraries/defaultFieldKinds.spec.ts

```diff
@@ -49,6 +50,7 @@ const tree1ContextuallyTyped: ContextuallyTypedNodeDataObject = {
 // Therefore it should not be using JsonableTrees.
 // The usages of this (and other JsonableTrees) such as ValueChangeset should be changed to use
 // a tree format intended for in memory use, such as Cursor or MapTree.
+// TODO: Figure out issue with deepfreezing here
```

Comment:
What issue exactly?

Reply:
So I talked to Craig about this issue, and he pointed out that our cursor APIs tend make it so that a consumed cursor ends up back where it started. That indeed seems to be the case our usages of cursors when encoding changesets or when consuming deltas.

He advised us to be more selective about what we freeze and that if we want to ensure specific actions are not taken on your test data, mocks might be a better approach.

That said, he seemed to like the idea of passing around something like TreeChunk instead since it is inherently safer.
So it may make sense to wait for that practice to become more standardized.